src/app/auth/switch-org/switch-org.page.ts

```
@@ -186,11 +186,14 @@ export class SwitchOrgPage implements OnInit, AfterViewChecked {
     const orgId = that.activatedRoute.snapshot.params.orgId as string;
     const txnId = this.activatedRoute.snapshot.params.txnId as string;
     const openSMSOptInDialog = this.activatedRoute.snapshot.params.openSMSOptInDialog as string;
+    const openVirtualCards = this.activatedRoute.snapshot.params.openVirtualCards as string;
 
     if (orgId && txnId) {
       return this.redirectToExpensePage(orgId, txnId);
     } else if (openSMSOptInDialog === 'true' && orgId) {
       return this.redirectToDashboard(orgId);
+    } else if (orgId && openVirtualCards) {
+      return this.redirectToCorporateCards(orgId, openVirtualCards === 'true');
```

Biome (2.3.14)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)